src/modules/auth/entities/account-refresh-token.entity.ts

```diff
@@ -0,0 +1,38 @@
+import { Account } from '#/modules/user/entities/account.entity';
+import {
+  Entity,
+  PrimaryGeneratedColumn,
+  Column,
+  ManyToOne,
+  CreateDateColumn,
+  JoinColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+
+@Entity('account_refresh_token')
+export class AccountRefreshToken {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column()
+  token: string;
+
+  @ManyToOne(() => Account, { onDelete: 'CASCADE' })
+  @JoinColumn({ name: 'account_id' })
+  user: Account;
```

Comment:
Be consistent with naming. Account instead of user

Reply:
Done